packages/api-gateway/src/drivers/http/adapters/places/handlers.js

```diff
@@ -1,17 +1,17 @@
 async function postPlace(req, reply) {
   const {
-    place_name, price_per_night_usd, host_id, type,
+    placeName, pricePerNightUsd, hostId, type,
```

Comment:
Places squad are using underscore case to name their variables, so this will cause errors, use aliases instead to avoid errors and comply with the current ESLint rules.
```suggestion
    place_name: placeName,
    price_per_night_usd: pricePerNightUsd,
    host_id: hostId,
    type,
```